libobs/util/base.c

```diff
@@ -83,6 +83,15 @@ void base_set_log_handler(log_handler_t handler, void *param)
 
 void base_set_crash_handler(void (*handler)(const char *, va_list, void *), void *param)
 {
+	static bool non_default_handler_set = false;
+
+	if (non_default_handler_set) {
+		blog(LOG_ERROR, "Tried to set a crash handler when one already exists. "
+				"This is unsupported.");
+		return;
+	}
+
+	non_default_handler_set = true;
```

Comment:
Have you checked if using a `pthread_once` pattern is the better choice here? I dunno if `base_set_crash_handler` is strictly called by only a single thread, and `pthread_once` should ensure that only the first call by _any_ thread sets the value with static storage.